packages/office-ui-fabric-react/src/components/Persona/PersonaPresence/PersonaPresence.base.tsx

```diff
@@ -67,7 +68,7 @@ export class PersonaPresenceBase extends BaseComponent<IPersonaPresenceProps, {}
     }
 
     return (
-      <div className={classNames.presence} style={coinSizeWithPresenceStyle}>
+      <div className={classNames.presence} style={coinSizeWithPresenceStyle} {...presenceTitle && { title: presenceTitle }}>
```

Comment:
>title [](start = 100, length = 5)

Is the title attribute accessible with screen readers?

Reply:
no

Reply:
Hm, that is unfortunate. We should attempt to make this available for screen readers as well. We could maybe set an aria-label in addition to the title?

Reply:
Amm, as per the current behavior, the whole of the PersonaCoin is not being read out by the screen reader. So adding aria-label won't make any difference.

Reply:
You can still navigate to the element though with the screenreader (not using tabbing)

Reply:
How were you able to navigate to the element through the screenreader without tabbing? I tried Virtual cursor mode with JAWS but was not able to navigate to PersonaCoin.

Reply:
I am not that familiar with JAWS, but you can see instructions for Narrator on how to use read mode to read text [here](https://support.microsoft.com/en-us/help/22799/windows-10-narrator-reading-text). JAWS does have similar functions

@Markionium - Ping :)

Reply:
Ah, thanks. I tested with Narrator. It reads out presenceTitle when present. So no additional change needed for this. :)

Reply:
Cool!